decidim-pages/app/views/decidim/pages/application/show.html.erb

```diff
@@ -1,5 +1,4 @@
 <% add_decidim_meta_tags({
-  title: translated_attribute(@page.title),
```

Comment:
Where is the title actually set? Does this also happen with `decidim_page.html.erb`?